gradle/libs.versions.toml

```diff
@@ -0,0 +1,28 @@
+[versions]
+agp = "8.11.0"
+androidTools = "30.0.0" # Update this values in sync with agp.
+kotlin = "2.2.0"
+autoService = "1.1.1"
+minSdk = "14"
+compileSdk = "30"
```

Comment:
These can be configured in the root build file or something. But they're fine here for now.

Reply:
Yeah, want to keep things simple. Will update it in the next PR.